### ISSUE TITLE
Emerson?

### ISSUE THREAD

**Issue by edysegura** [rodolfodsfaria/car-parking-exame#1]
Olá Rodolfo!

Por que o seu commit está com o nome do Emerson?

![image](https://cloud.githubusercontent.com/assets/5324414/11945899/d82b00f4-a83b-11e5-8f4f-dd03531cde9e.png)

![image](https://cloud.githubusercontent.com/assets/5324414/11946203/57bb96be-a83f-11e5-986c-e5ec07dacc97.png)

**Comment by edysegura** [comment]
@emersonw Você ajudou o Rodolfo a fazer a prova?

**Comment by ghost** [comment]
Sim, fiz o commit no github do meu computador, porque no dele, não deu
certo. Também passei meu github pra ele estudar.
Em 21/12/2015 11:44 PM, "Edy Segura" notifications@github.com escreveu:

**Comment by edysegura** [comment]
@emersonw Ele te chamou e aí você fez o commit pare ele? Vocês estavam fazendo em dupla?
@rodolfodsfaria me procure o quanto antes, até o momento sua avaliação está inválida.

**Comment by rodolfodsfaria** [comment]
Bom dia Eddy, eu já havia pegado o projeto do Emerson antes da prova para estudar, você pode verificar através do meu github https://github.com/rodolfodsfaria/prova/tree/master/prova. 

Sobre o commit, não estava conseguindo fazer por aqui, estava dando erro. Então pedi ao Emerson para fazer o commit pra mim do computador dele.
